Checkers/Board.cs

```diff
@@ -0,0 +1,272 @@
+                    new int [] {6,3},
+                    new int [] {6,5},
+                    new int [] {6,7},
+                    new int [] {7,0},
+                    new int [] {7,2},
+                    new int [] {7,4},
+                    new int [] {7,6},
+            };
+
+            for(int i = 0; i < whitePositions.Count; i++)
+            {
+
+                Checkers.Add(new Checker("white", whitePositions[i]));
+
+            }
+
+             for(int i = 0; i < blackPositions.Count; i++)
+            {
+                Checkers.Add(new Checker("black", blackPositions[i]));
+            }
+
+            //validates existence of checkers in collection
+            // foreach(Checker c in Checkers)
+            // {
+            //     Console.WriteLine(c.Symbol);
+            // }
+   
+        }
+        
+        // places the checkers on the grid from the List
+        public void PlaceCheckers()
+        {
+            for (var i = 0; i < Checkers.Count; i++)
+            {
+                int[] position = Checkers[i].Position;
+                this.Grid[position[0]][position[1]] = Checkers[i].Symbol;
+            }
+        }
+
+
+        
+        
+        //draws the board based on position of checkers with a virtual board from a jagged array
+        public void DrawBoard()
+        {
+            CreateBoard();
+            GenerateCheckers();
+            PlaceCheckers();
+            
+
+            for(int i = 0 ; i < 8 ; i++)
+            {
+                Console.Write(i.ToString());
+                for(int j = 0; j < 8; j++)
+                {
+                    Console.Write( "|" + this.Grid[i][j]);
+                }
+                Console.Write("|");
+                Console.WriteLine("\r");              
+
+            }
+            
+        }
+
+        // used to refresh board after moves/removes
+        public void reloadBoard()
+        {
+            CreateBoard();
+            PlaceCheckers();
+            for(int i = 0 ; i < 8 ; i++)
+            {
+                Console.Write(i.ToString());
+                for(int j = 0; j < 8; j++)
+                {
+                    Console.Write( "|" + this.Grid[i][j]);
+                }
+                Console.Write("|");
+                Console.WriteLine("\r");
+                
+            }
+
+        }
+        
+        // used to return checker class object found based on coordinates
+        public Checker SelectChecker(int row, int column)
+        {
+            return Checkers.Find(x => x.Position.SequenceEqual(new List<int> { row, column }));
+        }
+
+
+        // main method invoked for input and moving checkers no board
+        public void moveCheckerOnBoard()
+        {
+
+            int oldRow = 0;
+            int oldColumn = 0;
+            int newRow = 0;
+            int newColumn = 0;
+
+            Console.WriteLine("Enter pick up row.");
+            oldRow = int.Parse(Console.ReadLine());
```

Comment:
use `int.TryParse(...)` instead. If I enter `A` this  will crash your application.